docs/framework/guides/tutorials/implementing-a-block-widget.md

```diff
@@ -814,11 +814,11 @@ export default class SimpleBoxEditing extends Plugin {
 	}
 
 	_defineSchema() {
-		// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.